[typo](docs)supplement the document content

yagagagaga opened the pull request:
# Proposed changes

Issue Number: close #xxx

## Problem summary

The original document is too simple, so I made some additions.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

yagagagaga committed:
[typo](docs)supplement the document content

luzhijing commented on docs/zh-CN/docs/sql-manual/sql-functions/aggregate-functions/grouping.md:
```diff
@@ -25,25 +25,82 @@ under the License.
-#### Syntax
 
-`GROUPING(expr)`
+用在含有 CUBE、ROLLUP 或 GROUPING SETS 的SQL语句中，用于表示进行 CUBE、ROLLUP 或 GROUPING SETS 操作的列是否汇总。当结果集中的数据行是 CUBE、ROLLUP 或 GROUPING SETS 操作产生的汇总结果时，该函数返回 1，否则返回 0。GROUPING 函数可以在 `SELECT`、`HAVING` 和 `ORDER BY` 子句当中使用。
```
add space after and before English letters in CN docs

yagagagaga replied:
I have made corrections.

luzhijing commented on docs/zh-CN/docs/sql-manual/sql-functions/aggregate-functions/grouping.md:
```diff
@@ -25,25 +25,82 @@ under the License.
-#### Syntax
 
-`GROUPING(expr)`
+用在含有 CUBE、ROLLUP 或 GROUPING SETS 的SQL语句中，用于表示进行 CUBE、ROLLUP 或 GROUPING SETS 操作的列是否汇总。当结果集中的数据行是 CUBE、ROLLUP 或 GROUPING SETS 操作产生的汇总结果时，该函数返回 1，否则返回 0。GROUPING 函数可以在 `SELECT`、`HAVING` 和 `ORDER BY` 子句当中使用。
```
add space before and after English letters in CN docs

yagagagaga replied:
I have made corrections.

luzhijing reviewed (commented):
Some formatting issues

yagagagaga committed:
Update grouping.md

Add space before and after English letters in CN docs and keep the English case consistent.

yagagagaga commented on docs/zh-CN/docs/sql-manual/sql-functions/aggregate-functions/grouping.md:
````diff
@@ -25,25 +25,82 @@ under the License.
+  ROLLUP(camp); -- CUBE(camp) 和 GROUPING SETS((camp)) 同样也有效;
+```
+
+结果集在 `camp` 列下有两个 NULL 值，第一个 NULL 值表示 `ROLLUP` 操作的列的汇总结果，这一行的 `occ_cnt` 列表示所有 `camp` 的 `occupation` 的计数结果，在 `grouping` 函数中返回 1。第二个 NULL 表示 `camp` 列中本来就存在的 null 值。
````
ok

yagagagaga committed:
Update grouping.md

Change the Chinese title to English

luzhijing reviewed (approved):
LGTM
